.docker/scripts/serve.sh

```diff
@@ -15,7 +15,7 @@ done
 
 . ${APP_DIR}/scripts/activate
 if (which ckan > /dev/null); then
-    ckan -c ${CKAN_INI} run -r
+    ckan -c ${CKAN_INI} run -r -t
```

Comment:
we may want to enable 4 processes as well as threads to see if that makes it go faster.
-r == --disable-reloader
-t ==  --threaded
ckan -c ${CKAN_INI} run --disable-reloader --threaded --processes 4

Reply:
scratch that, it is either threaded or processes based. Can't have both